spec.html

```diff
@@ -4096,6 +5071,15 @@ <h1>NumberToString ( _m_ )</h1>
             <a href="http://netlib.sandia.gov/fp/g_fmt.c">http://netlib.sandia.gov/fp/g_fmt.c</a> and may also be found at the various `netlib` mirror sites.</p>
         </emu-note>
       </emu-clause>
+
+      <emu-clause id="sec-biginttostring" aoid="BigIntToString">
+        <h1>BigIntToString ( _i_ )</h1>
+        <p>The abstract operation BigIntToString converts a BigInt _i_ to String format as follows:</p>
+        <emu-alg>
+          1. If _i_ is less than zero, return the string-concatenation of the String `"-"` and ToString(-_i_).
```

Comment:
should `ToString` here be `BigInt::ToString` or `BigIntToString`?

Reply:
f963efc should have replaced `BigIntToString` with `BigInt::ToString`.

Reply:
Oops. Probably I reverted this after rebase. I'll change it to `BigInt::ToString`.